.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = ../../arfc/squirrel
 [submodule "moose"]
 	path = moose
-	url = ../../idaholab/moose
+	url = https://github.com/idaholab/moose
```

Comment:
The reason for the `../../` is to use HTTPS for the submodule clone if that was the way the repository was cloned, and SSH if that was the way. Did you have an issue that this fixed?

Reply:
No, this was unintentional. I was building Zapdos and using an installation of moose I had in `~/projects/moose` rather than as a submodule. When I was doing this I removed the moose directory locally and then re-added it as a submodule so it wasn't removed for the PR. I have corrected this error in my most recent commit.

Reply:
Ah ok, thanks for fixing it back up. Just in case you didn't know, you can set `MOOSE_DIR` in your environment or on the same command line right before typing `make` and Zapdos will use whatever copy of MOOSE you want it to, whether the submodule is checked out or not. It's nice when you're testing something in your regular MOOSE copy and can't be bothered to set the submodule to the test commit. 😄

Reply:
Thanks for letting me know! I will use that in the future.